gui/qt/main_window.py

```diff
@@ -2681,6 +2680,9 @@ def read_tx_from_qrcode(self):
             self.print_error("Warning: QR dialog is already presented, ignoring.")
             return
         from electroncash import get_config
+        from .qrreaderutil import warn_unless_can_import_qrreader
+        if not warn_unless_can_import_qrreader():
```

Comment:
As per my comment, here you would need to pass `self` to the warn function (see the `warn` style function in `__init__.py` for an illustration). 

 Document the `warn_unless_can_import_qrreader` that it accepts only a `MessageBoxMixin` child as a paremeter.